lib/feature_flags/adapters/null_adapter.rb

```diff
@@ -0,0 +1,16 @@
+# typed: strict
+# frozen_string_literal: true
+
+module FeatureFlags
+  module Adapters
+    class NullAdapter < Adapter
+      extend T::Sig
+
+      sig { override.params(account_request: Params::AccountRequest).returns(T::Boolean) }
+      def feature_account_enabled?(account_request)
+        account_request.feature.empty?
```

Comment:
Is it supposed to have this line if we want to return always true?

Reply:
whoopsy